test/inductor/test_torchinductor.py

```diff
@@ -8470,23 +8474,87 @@ def return_true(*args, **kwargs):
             # should_pad_bench always returns False if has_triton returns False
             self.assertFalse(should_pad)
 
-    def test_bessel_j0(self):
-        def fn(x):
-            return torch.special.bessel_j0(x)
+    @parametrize(
+        "name, op",
+        [
+            subtest((name, getattr(torch.special, name)), name=name)
+            for name in torch.special.__all__
+            if "_polynomial_" not in name
+        ],
+    )
+    def test_pointwise(self, name, op):
+        dtype = torch.float32
+
+        if self.device == "cuda" and name in {
+            "airy_ai",
+            "bessel_y0",
+            "bessel_y1",
+            "erfcx",
+            "gammainc",
+            "gammaincc",
+            "i1",
+            "i1e",
+            "modified_bessel_i1",
+            "modified_bessel_k0",
+            "modified_bessel_k1",
+            "ndtri",
+            "scaled_modified_bessel_k0",
+            "scaled_modified_bessel_k1",
+            "spherical_bessel_j0",
+            "zeta",
+        }:
+            # <func>_cuda not implemented for Half
+            dtype = torch.float64
```

Comment:
You can use `check_lowp=False` in `self.common` to skip the half-precision check.